BackendApartmentReservation/BackendApartmentReservation/Apartments/ApartmentsController.cs

```diff
@@ -32,6 +33,14 @@ public async Task<IEnumerable<string>> Get()
             return await Task.FromResult(new[] { "flat1", "flat2" });
         }
 
+        [HttpPost]
+        [Route("apartments/create")]

```

Comment:
POST says its a create

Generally we dont use verbs (at least create, update etc) in REST endpoints